Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -3375,18 +3375,22 @@ def get_power_query_output(cmd: str, interval: int, timeout: int, args: dict, qu
     if query_response.get("status") == "FINISHED" and query_response.get("progress") == 100:
         headers = get_columns_from_result(query_response.get("columns", []))
         context_entries = [dict(zip(headers, row)) for row in query_response.get("data", [])]
-        readable_text = f"SentinelOne - Get Power Query Results - for ID {query_response.get('queryId', '')}"
+        readable_text = f"SentinelOne - Get Power Query Results for ID {query_response.get('queryId', '')}"
         recommendations = query_response.get("recommendations", [])
         if recommendations and len(recommendations) >= 1:
             recommendation = recommendations[0]
             readable_text += f"\nRecommendation: {str(recommendation)}"
         return CommandResults(
-            readable_output=tableToMarkdown(readable_text, context_entries, headers=headers, removeNull=True),
-        )
+            readable_output=tableToMarkdown(readable_text, context_entries, removeNull=True,
+                                            metadata='\nSummary information and details about the power query',
+                                            headerTransform=pascalToSpace),
+            outputs_prefix='SentinelOne.PowerQuery',
+            outputs=context_entries,
+            raw_response=query_response)
     else:
         scheduled_command = ScheduledCommand(command=cmd, next_run_in_seconds=interval, args=args, timeout_in_seconds=timeout)
-        command_results = CommandResults(scheduled_command=scheduled_command)
-        return command_results
+        return_results(CommandResults(scheduled_command=scheduled_command))
```

Comment:
```suggestion
        return CommandResults(scheduled_command=scheduled_command)
```
You're already calling the return_results() in the main function for this command